support-services/src/main/scala/com/gu/aws/AwsCloudWatchMetricSetup.scala

```diff
@@ -15,9 +15,12 @@ object AwsCloudWatchMetricSetup {
   def catalogFailureRequest(environment: TouchPointEnvironment): MetricRequest =
     getMetricRequest(MetricName("CatalogLoadingFailure"),
       Map(
-        MetricDimensionName("Environment") -> MetricDimensionValue(environment.toString)
+        MetricDimensionName("Environment") -> MetricDimensionValue(environment.envValue)
       ))
 
+  def revenueDistributionFailureRequest: MetricRequest =
+    getMetricRequest(MetricName("RevenueDistributionFailure"), Map())
```

Comment:
we would need a Stage -> PROD/CODE/DEV in the map as well, so it doesn't alarm when people are testing

Reply:
It will only run for PROD support-workers so it won't be a problem for DEV or CODE, we might get some test user failures but it would involve quite a lot of code change to avoid that so I'd like to leave it as is for now and only do that work if it becomes a problem

Reply:
You're right, I had thought that because we were only creating the alarm for the PROD stage we wouldn't need it but of course that's just the alarm, the metric needs to have the stage as well. I've added this now.

Reply:
yes, and from your comment I guess you edited it, I agree about test users ideally shouldn't trigger this, but we get alarms for test users already as illustrated by the gifting testing session, that's one for another debate I think.